app/styles/mixins.less

```
@@ -124,5 +124,11 @@
 }
 
 .bg-active(@color: @black) {
-  background-color: rgba(@color, .2);
+  background-color: rgba(@color, .25);
+}
+
+.text-ellipsis {
```

mixinは引数リストをとるように定義してください。構文的にCSSとしてclassを生成すべきなのか、利用されていなければ削除してよいのかLessのコンパイラが判別できず、 **全てのmixins.less参照箇所でそれぞれにclassが生成されます**
see. https://github.com/n-air-app/n-air-app/pull/236
```suggestion
.text-ellipsis() {
```

---

こちら確認しました！
以降は引数をとるように定義します🙇‍♂️